src/wallet/rpcwallet.cpp

```diff
@@ -152,6 +152,11 @@ UniValue getnewaddress(const JSONRPCRequest& request)
     return CBitcoinAddress(keyID).ToString();
 }
 
+void DeleteAccount(CWallet * const pwallet, std::string strAccount)
```

Comment:
Nit, shouldn't we follow the convention in new code in favor of consistency? In that case rename `strAccount` to `account_name` for instance?